events/wc_cenarion_circle_events.txt

```diff
@@ -205,6 +208,25 @@ character_event = {
 		move_character = FROM
 	}
 }
+character_event = {
+	id = WCCC.23
+
+	is_triggered_only = yes
+
```

Comment:
What if the Kaldorei doesn't exists or is ruled by demons?
```suggestion

	trigger = {
		e_kaldorei = {
			has_holder = yes
			holder_scope = {
				is_untouchable_trigger = no
				can_rule_peacefully_trigger = yes
				religion_group = druidism_group
			}
		}
	}
	
```

Reply:
fixed